suite-native/react-native-graph/src/AnimatedLineGraph.tsx

```diff
@@ -355,11 +355,18 @@ export function AnimatedLineGraph<TEventPayload extends object>({
         (fingerX: number) => {
             const fingerXInRange = Math.max(fingerX - horizontalPadding, 0);
 
+            const date = pointsInRange[pointsInRange.length - 1];
+
+            // If there was graph error, points in range can be empty
+            if (!date) {
+                return;
+            }
+
             const index = Math.round(
                 (fingerXInRange /
                     getXInRange(
                         drawingWidth,
-                        pointsInRange[pointsInRange.length - 1]!.date,
+                        pointsInRange[pointsInRange.length - 1].date,
```

Comment:
You can use `lastDate` const here

Reply:
Yeah, that's the whole point of why I created the const :D Then I forgot, good catch 
https://github.com/trezor/trezor-suite/pull/11495/commits/b2a360134ec65b9942c730967614b04651b13c9b